### PR TITLE
[Refactor] Refactor PrivType and ObjectType and unify the type format

### PR DESCRIPTION
**Pull request by HangyuanLiu**
Signed-off-by: HangyuanLiu <460660596@qq.com>

## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [x] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

**Comment by HangyuanLiu** [comment]
run starrocks_fe_unittest

**Review by Astralidea — APPROVED** [review]

File: fe/fe-core/src/main/java/com/starrocks/privilege/PrivilegeType.java [hunk 1]
```diff
@@ -14,33 +14,46 @@
...
+    @SerializedName("i")
+    private final int id;
+
+    PrivilegeType(int id) {
+        this.id = id;
+    }
+
+    public int getId() {
+        return id;
+    }
```
Comment (gengjun-git) [inline fe/fe-core/src/main/java/com/starrocks/privilege/PrivilegeType.java:58]: Is this change compatible for upgrade?

Comment (HangyuanLiu) [reply fe/fe-core/src/main/java/com/starrocks/privilege/PrivilegeType.java:58]: Yes, there is no compatibility issue with upgrading from older versions. Old versions are not stored according to PrivilegeType

**Comment by HangyuanLiu** [comment]
run starrocks_fe_unittest

**Review by gengjun-git — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 274 / 349 (78.51%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/http/rest/GetLoadInfoAction.java|0|1|00.00%|[82]|
|:large_blue_circle:|com/starrocks/qe/ExecuteAsExecutor.java|0|1|00.00%|[42]|
|:large_blue_circle:|com/starrocks/sql/analyzer/AnalyzerUtils.java|0|1|00.00%|[158]|
|:large_blue_circle:|com/starrocks/qe/ShowExecutor.java|5|15|33.33%|[1887, 1897, 1922, 1923, 1924, 1953, 1961, 1986, 1987, 1988]|
|:large_blue_circle:|com/starrocks/sql/analyzer/AstToStringBuilder.java|3|8|37.50%|[328, 329, 338, 388, 457]|
|:large_blue_circle:|com/starrocks/privilege/PrivilegeManager.java|75|120|62.50%|[222, 261, 657, 658, 659, 660, 661, 662, 685, 686, 687, 688, 689, 690, 883, 884, 885, 886, 887, 889, 898, 899, 914, 915, 916, 917, 918, 920, 992, 993, 994, 1044, 1047, 1072, 1073, 1074, 1075, 1076, 1078, 1386, 1387, 1388, 1389, 1390, 1393]|
|:large_blue_circle:|com/starrocks/sql/ast/GrantPrivilegeStmt.java|3|4|75.00%|[41]|
|:large_blue_circle:|com/starrocks/sql/analyzer/PrivilegeStmtAnalyzerV2.java|31|36|86.11%|[184, 246, 247, 260, 280]|
|:large_blue_circle:|com/starrocks/mysql/privilege/AuthUpgrader.java|44|47|93.62%|[885, 970, 971]|
|:large_blue_circle:|com/starrocks/privilege/DefaultAuthorizationProvider.java|40|43|93.02%|[153, 210, 220]|
|:large_blue_circle:|com/starrocks/privilege/ActionSet.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/analyzer/PrivilegeCheckerV2.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/privilege/PrivilegeType.java|30|30|100.00%|[]|
|:large_blue_circle:|com/starrocks/privilege/PrivilegeCollection.java|18|18|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/ast/BaseGrantRevokePrivilegeStmt.java|4|4|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/ast/RevokePrivilegeStmt.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/privilege/RolePrivilegeCollection.java|3|3|100.00%|[]|
|:large_blue_circle:|com/starrocks/privilege/ObjectType.java|12|12|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/analyzer/PrivilegeStmtAnalyzer.java|2|2|100.00%|[]|

**Pull request merged**